documentation/src/main/markdown/properties/detectors/overview.md

```diff
@@ -1,3 +1,9 @@
 # Detector Properties
 
 This section contains details about properties that configure the [solution_name] detectors.
+
+[solution_name] uses detectors to find and extract dependencies from supported package managers.
+
+By default, all detectors are eligible to run. The set of detectors that actually run depends on the files that exist in your project directory. 
+
+<note type="note">Low accuracy detectors, see [Detector search and accuracy](../../runningdetect/detectorcascade.md), do not support configuration properties.</note>
```

Comment:
@sig-cpotts Based on what Karolyn mentioned in the email thread, this is not correct. - _"There are detectors that are essentially the Project Inspector tool (such as NugGet Project Inspector) that do not get passed all the Detect properties. There are specific detect.project.inspector.* properties that do directly affect them, however.
 
Other low accurancy detectors, such as NPM Package Json Parse and Gemspec Parse, do honor Detect properties."_

Reply:
Yep @dmamidisynopsys - I've replied in that thread. Some further consideration is required on this one. Thanks.